src/main/scala/RPS.scala

```diff
@@ -0,0 +1,56 @@
+object RPS {
+
+  def main (args:Array[String]): Unit ={
+    playerMove()
+  }
+  def pickWinner(move1:String, move2:String)=(move1,move2) match {
```

Comment:
These lines could be separated a bit to improve readability

Reply:
Eg Could have `(move1.toLowerCase(), move2.toLowerCase() match`, then you wouldn't need `.equalsIgnoreCase` for each option